src/test/java/com/galvanize/alpha/speedwaytrials/racecars/RaceCarIT.java

```diff
@@ -50,7 +51,54 @@ public void addCars() throws Exception {
         mockMvc.perform(get("/cars"))
             .andExpect(status().isOk())
             .andExpect(jsonPath("length()").value(1))
-            .andExpect(jsonPath("[0].model").value("Ferrari"))
-            .andDo(document("GetCars"));
+            .andExpect(jsonPath("[0].model").value("Corvette"))
+            .andExpect(jsonPath("[0].nickname").value("The Condor"))
+            .andExpect(jsonPath("[0].year").value(2019))
+            .andExpect(jsonPath("[0].status").value("AVAILABLE"))
+            .andExpect(jsonPath("[0].topSpeed").value(189))
+            .andDo(document("GetCars", responseFields(
+                fieldWithPath("[0].model").description("Corvette")
+                ,fieldWithPath("[0].nickname").description("The Condor")
+                ,fieldWithPath("[0].year").description(2019)
+                ,fieldWithPath("[0].status").description("AVAILABLE")
+                ,fieldWithPath("[0].topSpeed").description(189)
+            )));
+    }
+
+    @Test
+    public void multipleCars() throws Exception{
+        CarsDto carsDto1 = new CarsDto("Corvette", "The Condor",2019,"AVAILABLE",189);
+        CarsDto carsDto2 = new CarsDto("Ferrari", "Breadvan",2020,"AVAILABLE",193);
+
+        mockMvc.perform(post("/cars").content(objectMapper.writeValueAsString(carsDto1)).contentType(MediaType.APPLICATION_JSON)).andExpect(status().isCreated()).andDo(document("AddCars"));
+        mockMvc.perform(post("/cars").content(objectMapper.writeValueAsString(carsDto2)).contentType(MediaType.APPLICATION_JSON)).andExpect(status().isCreated()).andDo(document("AddCars"));
+        mockMvc.perform(get("/cars"))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("length()").value(2))
+            .andExpect(jsonPath("[0].model").value("Corvette"))
+            .andExpect(jsonPath("[0].nickname").value("The Condor"))
+            .andExpect(jsonPath("[0].year").value(2019))
+            .andExpect(jsonPath("[0].status").value("AVAILABLE"))
+            .andExpect(jsonPath("[0].topSpeed").value(189))
+            .andDo(document("GetCars", responseFields(
+                fieldWithPath("[0].model").description("Corvette")
```

Comment:
You can actually use `[].model` for the path and it will match appropriately.

You can even document `[]` as the array if you wanted.